core_backend/app/auth/config.py

```diff
@@ -1,15 +1,5 @@
 import os
 
-# user 1
```

Comment:
I moved all of this to the main `app.config.py` so it can be read by the user-loading script (which is a stand-in for until we have OAuth-aware user creation) and pytest, but Auth doesnt need to know about this anymore since it reads from the DB